packages/rocketchat-lib/server/models/Subscriptions.js

```diff
@@ -11,7 +11,19 @@ class ModelSubscriptions extends RocketChat.models._Base {
 		this.tryEnsureIndex({ 'u._id': 1, 'name': 1, 't': 1, 'code': 1 }, { unique: 1 });
 		this.tryEnsureIndex({ 'open': 1 });
 		this.tryEnsureIndex({ 'alert': 1 });
-		this.tryEnsureIndex({ 'unread': 1 });
```

Comment:
running the `$inc` command on `unread`  field causes the index to be fully rebuilt at least once on each `update` (but maybe the index was rebuilt multiple times because we are updating multiple records at once), so removing the index (which is not being used by any heavy `find` command) helped increasing performance (the `$inc` update is now 50% faster)